### PR TITLE
Use "web-search" keyboardType on iOS

### LINKED ISSUE #1468: iOS url input should use url keyboard
iOS provides different keyboards for different contexts. For example, browser URL bars usually include a period Next to the space bar. Ours does not. We should be able to change this with a simple configuration flag.

0E07BF98-4A53-47A4-84BF-B8C06CEEDA12

2BC0AF93-425F-4751-AE8D-517188CA6FF8

### PR DESCRIPTION
**Pull request by rickycodes**
**Description**

re: #1468 this _should_ now use the `url` keyboard type on iOS for the url modal. I don't really have a way to test this myself, but it should be good.

**Checklist**

* [x] There is a related GitHub issue
* [x] Tests are included if applicable
* [x] Any added code is fully documented

**Issue**

Resolves #1468

### CODE REVIEW

File: app/components/Views/BrowserTab/index.js [hunk 1]
```diff
@@ -1625,6 +1625,7 @@ export class BrowserTab extends PureComponent {
 			>
 				<View style={styles.urlModalContent} testID={'url-modal'}>
 					<TextInput
+						keyboardType={Device.isIos() ? 'url' : 'default'}
```
Comment (rickycodes) [inline app/components/Views/BrowserTab/index.js:1628]: I could likely get away with just "url" vs using the ternary :thinking:

**Review by estebanmino — COMMENTED** [review]
I'd have to run it but I'm almost sure this won't allow spaces on the keyboard, something we need to do a search cc @ibrahimtaveras00

**Comment by rickycodes** [comment]
ah, good point @estebanmino , I hadn't considered that. I guess since we're using this as an omnibar (vs just for urls) we might be able to just close the issue

**Comment by ibrahimtaveras00** [comment]
@estebanmino your fears/concerns were correct, the space bar is now missing.

Screen Shot 2020-05-06 at 3 00 52 PM

@rickycodes I am in agreement to possibly just close the issue since it's an omnibar and we want to give users the ability to search non URL text which will most likely include using the space bar from time to time.

Unless we use a similar keyboard to what Safari and Chrome use 

Screen Shot 2020-05-06 at 3 06 28 PM

**Review by ibrahimtaveras00 — APPROVED** [review]
update looks good, QA Passed 👍

**Pull request merged**